internal/install/application_configuration.go

```diff
@@ -0,0 +1,95 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package install
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+)
+
+// ApplicationConfiguration represents the configuration of the elastic-package.
+type ApplicationConfiguration struct {
+	c configFile
+}
+
+type configFile struct {
+	Stack stack `yaml:"stack"`
+}
+
+type stack struct {
+	ImageRefOverrides map[string]ImageRefs `yaml:"imageRefOverrides"`
```

Comment:
Should we switch the yaml setting to snake case? Looking at other settings in package yaml files used for configuration (e.g. pipeline test case configuration), we use snake case.

Reply:
Fixed!